app-drawer-layout/app-drawer-layout.html

```diff
@@ -243,53 +149,36 @@
        * @property drawer
        */
       get drawer() {
-        return Polymer.dom(this.$.drawer).getDistributedNodes()[0];
+        return Polymer.dom(this.$.drawerContainer).getDistributedNodes()[0];
       },
 
       _tapHandler: function(e) {
         var target = Polymer.dom(e).localTarget;
         if (target && target.hasAttribute('drawer-toggle')) {
-          this.toggleDrawer();
+          this.drawer.toggle();
         }
       },
 
-      _drawerOpened: function(narrow, opened) {
-        return !narrow || opened;
+      _responsiveChange: function(narrow) {
+        this._setNarrow(narrow);
+        this._resetLayout();
       },
 
-      _drawerActivate: function() {
-        if (!this.disableDrawerActivate && this.narrow) {
-          this.async(function() {
-            this.opened = false;
-          });
+      _resetLayout: function() {
```

Comment:
Should listen for `<app-drawer>` app-drawer-position-changed event and update marginLeft/Right